lib/Braintree/Http.php

```diff
@@ -166,9 +167,16 @@ public function _doUrlRequest($httpVerb, $url, $requestBody = null)
         curl_close($curl);
         if ($this->_config->sslOn()) {
             if ($httpStatus == 0) {
-                throw new Exception\SSLCertificate();
+                if ($error_code == 35) {
+                    throw new Exception\SSLCertificate($error, $error_code);
+                } else {
+                    throw new Exception\Connection($error, $error_code);
+                }
             }
+        } else if ($error_code) {
+            throw new Exception\Connection($error, $error_code);
```

Comment:
Gotcha. I don't believe the sdk ever operates without SSL on, even in development, but figured it was prudent to provide some error reporting there as well.